tasks/utils/emissions-utils.ts

```diff
@@ -188,12 +190,13 @@ export const deployL2BridgeRecipients = async (
 }
 
 export const deployDisperseForwarder = async (signer: Signer, hre: HardhatRuntimeEnvironment): Promise<DisperseForwarder> => {
+    const chain = getChain(hre)
+    const nexusAddress = resolveAddress("Nexus", chain)
+    const disperseAddress = resolveAddress("Disperse", chain)
     const mtaAddress = MTA.address
-    const constructorArguments = [mtaAddress]
+    const constructorArguments = [nexusAddress, disperseAddress, mtaAddress]
```

Comment:
I think `DisperseForwarder` is missing arguments on its constructor, please validate this.